docs/standard/analyzers/portability-analyzer.md

```diff
@@ -21,8 +21,8 @@ Want to make your libraries multi-platform? Want to see how much work is require
 
 ## New targets
 
-* [.NET Core](https://dotnetfoundation.org/net-core): Has a modular design, employs side-by-side, and targets cross-platform scenarios. Side-by-side allows you to adopt new .NET Core versions without breaking other apps.
-* [ASP.NET Core](https://dotnetfoundation.org/asp-net-core): is a modern web-framework built on .NET Core thus giving developers the same benefits.
+* [.NET Core](../../core/index.md): Has a modular design, employs side-by-side, and targets cross-platform scenarios. Side-by-side allows you to adopt new .NET Core versions without breaking other apps.
+* [ASP.NET Core](/aspnet/core): is a modern web-framework built on .NET Core thus giving developers the same benefits.
```

Comment:
Remove the colon.

Reply:
so should I remove from all entries? They all have colons.

Reply:
Nope